packages/@pollyjs/adapter-node-http/tests/integration/adapter-test.js

```diff
@@ -109,4 +113,30 @@ function commonTests(transport) {
 
     expect(nativeHash).to.equal(recordedHash);
   });
+
+  it('should be able to upload a file', async function() {
+    const url = `${protocol}//example.com/upload`;
+    const { server } = this.polly;
+    const requests = [];
+    const formData = new FormData();
+
+    server.post(url).intercept((req, res) => {
+      requests.push(req);
+      res.send(201);
+    });
+
+    formData.append(
+      'upload',
+      fs.createReadStream(path.resolve(__dirname, '../../package.json'))
+    );
+    const body = await getStream(formData);
+
+    await nativeRequest(transport, url, {
+      body,
+      headers: formData.getHeaders(),
+      method: 'POST'
+    });
+
+    expect(requests.length).toBe(1);
```

Comment:
I'm noticing `requests[0].body` is empty due to https://github.com/Netflix/pollyjs/blob/7174b25ef3e97e07d22abac1d25a0e38eb9585b1/packages/%40pollyjs/adapter-node-http/tests/utils/native-request.js#L16. Is there a better mechanism for testing this?

Reply:
It should be deleting it from the internal `opts` object, it shouldn't be empty because of `delete opts.body`.  Should not effect the `req` object you are pushing to the requests array on line 124

Reply:
Update: fixed typo above, should now make more sense :)